kod/object/active/flag.kod

```diff
@@ -709,7 +708,11 @@ messages:
          }
       }
 
+      % If setting the faction as part of a bulk process (i.e. recreate),
```

Comment:
Make this a function comment (the string that can go right after the function declaration) rather than burying it the code.